css/style.css

```diff
@@ -42,12 +42,89 @@ body {
   width: 100%;
```

Comment:
We need to give the container either a fixed width (ie 900px), percentage less than 100% (because it's just going to span the entire viewport instead of centering) or a max-width.